platform/platform-paxweb-jettyconfig/src/main/java/org/codice/ddf/security/session/HashSessionIdManager.java

```diff
@@ -69,14 +69,14 @@ public HashSessionIdManager(Random random) {
     /**
      * @return Collection of String session IDs
      */
-    public Collection<String> getSessions() {
+    public synchronized Collection<String> getSessions() {
```

Comment:
Any performance testing done to see if these impact the system? Other syncs I've seen so far seem to be in lower-volume areas (I hope), but this seems like it'll be hit quite often.